servant/src/Servant/API/Fragment.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE DataKinds          #-}
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE PolyKinds          #-}
+{-# LANGUAGE TypeOperators      #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+module Servant.API.Fragment (Fragment, Fragment') where
+
+import           Data.Typeable
+                 (Typeable)
+import           Servant.API.Modifiers
+
+-- | Lookup the value associated to the URI fragment
```

Comment:
Same here: you don't lookup anything now as far as I understand, so the doc is misleading.

Furthermore, the modifiers are not needed anymore, right?

Reply:
Fixed.